src/bento/dashboardTabData.js

```diff
@@ -1140,6 +1411,185 @@ export const tabContainers = [
+    tabIndex: '4',
+    selectableRows: true,
+    tableDownloadCSV: customStudyTabDownloadCSV,
+    downloadFileName: 'C3DC Studies Download',
+    tableMsg: {
+      noMatch: 'No Matching Records Found',
+    },
+    addFilesRequestVariableKey: 'treatment_ids',
+    addFilesResponseKeys: ['fileIDsFromList'],
+    addAllFilesResponseKeys: ['treatmentOverview', 'files'],
+    addAllFileQuery: GET_ALL_FILEIDS_FROM_DIAGNOSISTAB_FOR_ADD_ALL_CART,
+    addSelectedFilesQuery: GET_ALL_FILEIDS_DIAGNOSISTAB_FOR_SELECT_ALL,
+  },
+  {
+    name: 'Treatment Response',
+    dataField: 'dataTreatmentResponse',
+    api: GET_TREATMENT_RESPONSE_OVERVIEW_QUERY,
+    paginationAPIField: 'treatmentResponseOverview',
+    defaultSortField: 'treatment_response_id',
+    defaultSortDirection: 'asc',
+    count: 'numberOfTreatmentResponses',
+    fileCount: 'treatmentResponseFileCount',
+    dataKey: 'id',
+    tableID: 'treatment_response_tab_table',
+    toolTipText: 'Count of Treatment Response Record',
+    extendedViewConfig: {
+      pagination: true,
+      manageViewColumns: false,
+      download: true,
+      downloadButtonConfig: {
+        title: 'DOWNLOAD DATA',
+        cloudIcon: true,
+      },
+    },
+    columns: [
+      {
+        dataField: 'participant_id',
+        header: 'Participant ID',
+        display: true,
+        tooltipText: 'sort',
+        role: cellTypes.DISPLAY,
+      },
+      {
+        dataField: "treatment_response_id",
+        header: "Treatment Response ID",
+        display: true,
+        tooltipText: "sort",
+        role: "cellTypes.DISPLAY"
+      },
+      {
+        dataField: 'response',
+        header: 'Response',
+        display: true,
+        tooltipText: 'sort',
+        role: cellTypes.DISPLAY,
+      },
+      {
+        dataField: 'age_at_response',
+        header: 'Age At Response',
+        display: true,
+        tooltipText: 'sort',
+        role: cellTypes.DISPLAY,
+      },
+      {
+        dataField: "response_category",
+        header: "Response Category",
+        display: true,
+        tooltipText: "sort",
+        role: "cellTypes.DISPLAY"
+      },
+      {
+        dataField: "response_system",
+        header: "Response System",
+        display: true,
+        tooltipText: "sort",
+        role: "cellTypes.DISPLAY"
+      },
+      {
+        dataField: 'dbgap_accession',
+        header: 'dbGaP Accession',
+        display: true,
+        tooltipText: 'sort',
+        role: cellTypes.DISPLAY,
+        linkAttr: {
+          rootPath: 'https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=',
+        },
+        cellType: cellTypes.CUSTOM_ELEM,
+      },
+    ],
+    id: 'treatment_response_tab',
+    tabIndex: '4',
+    selectableRows: true,
+    tableDownloadCSV: customStudyTabDownloadCSV,
+    downloadFileName: 'C3DC Studies Download',
+    tableMsg: {
+      noMatch: 'No Matching Records Found',
+    },
+    addFilesRequestVariableKey: 'treatment_response_ids',
```

Comment:
these lines can be removed or commented out 1589-1591

Reply:
Resolved.